Vagrantfile

```diff
@@ -0,0 +1,35 @@
+Vagrant.configure("2") do |config|
+	config.vm.box = "hashicorp/bionic64"
+
+	config.vm.network "forwarded_port", guest: 5000, host: 5000
+
+	config.vm.provision "shell", :privileged => false, inline: <<-SHELL
```

Comment:
We should add a brief comment in your Readme to advise new users how they can run the app in a VM